### ISSUE TITLE
A possible bug?

### ISSUE THREAD

**Issue by chainorato** [iqtree/iqtree2#180]
Hi,

I have reconstructed a tree using the pre-release IQ-TREE version 2.3.3.
I need to use this version since the constrained tree search and ModelFinder issue has just been fixed.
However, I got an error in the last line of the following.

```
Computing bootstrap consensus tree...
Reading input file NIM1K.trimal.splits.nex...
204 taxa and 412 splits.
Consensus tree written to NIM1K.trimal.contree
Reading input trees file NIM1K.trimal.contree
Log-likelihood of consensus tree: -33638.537
ERROR: Expecting ')', but found '/' instead (line 1 column 93)
```

Here is the code I used.

`iqtree2 -s NIM1K.trimal -g NIM1K.const.sp_tree.treefile -m MFP -B 1000 -alrt 1000 -T 1 -mem 16G`

Since there is no '/' in my input files, I don't know what exactly happened.
However, I cannot get the bootstrap and SH-Alrt values from the resulting tree (`.treefile`).

Could you figure out this issue?

Thank you in advance!

**Comment by thomaskf** [comment]
Hi @chainorato 
Thanks for reporting this. Can you send me the treefie and the alignment file so that we can check?
Many thanks,
Thomas

**Comment by chainorato** [comment]
Hi @thomaskf ,

Thank you so much for the prompt response.
Here are the alignment file and all results generated.
[NIM1K.tar.gz](https://github.com/iqtree/iqtree2/files/15106956/NIM1K.tar.gz)

Arnon

**Comment by thomaskf** [comment]
Hi Arnon
I can only find the output files from IQTREE, but cannot see the constraint-treefile "GeneID_167359_NIM1K.const.sp_tree.treefile" and the alignment file in the compressed file. 
Thanks,
Thomas

**Comment by chainorato** [comment]
My bad. Here it is.
[GeneID_167359_NIM1K.const.sp_tree.treefile.zip](https://github.com/iqtree/iqtree2/files/15108258/GeneID_167359_NIM1K.const.sp_tree.treefile.zip)

**Comment by thomaskf** [comment]
And the alignment file "GeneID_167359_NIM1K.trimal", please? Thanks.

**Comment by chainorato** [comment]
Here you are. Sorry for my carelessness
[GeneID_167359_NIM1K.trimal.zip](https://github.com/iqtree/iqtree2/files/15108301/GeneID_167359_NIM1K.trimal.zip)

**Comment by thomaskf** [comment]
Hi @chainorato ,
Thanks for reporting the issue and sending us the data file. The issue happened when running with the constraint tree feature with bootstrapping, and the input constraint tree file contains the internal node names. 
The issue has been fixed in the release version 2.3.4. I have tried with your command and the data set on my side, and the new version should work well.
Thank you and I close this issue now.
Thomas
